app/src/views/private/components/header-bar-actions.vue

```diff
@@ -84,7 +84,7 @@ export default defineComponent({
 
 @media (min-width: 960px) {
 	.actions .action-buttons .sidebar-toggle {
-		display: none;
+		display: none !important;
```

Comment:
This `!important` does not look to be required on my end (nor does this entire `.side-bar-toggle` block for that matter 😬 ). is this a workaround for a specific browser?
![image](https://user-images.githubusercontent.com/9389634/235453760-b800e6e5-04b0-40da-a3d7-167741e1a98d.png)

Reply:
Good eyes! It's a little unrelated fix, noticed while working on this feature. Easier to explain with the following videos 😄 

**Before**

https://user-images.githubusercontent.com/5363448/235456658-33a3121b-9177-460a-be92-f357c29337f4.mp4

**After**

https://user-images.githubusercontent.com/5363448/235456662-5cf5836c-0624-46ff-a98c-ba121bbc44ed.mp4